File: SoftUni_JAVA_BASIC_Jan_2023_SolutionsOfProblems/P05LAB_While_Loop/P07_MinNumber.java
package P05LAB_While_Loop;

import java.util.Scanner;

public class P07_MinNumber {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        int number = 0, minNumber = Integer.MAX_VALUE;
        while (!input.equals("Stop")) {
            number = Integer.parseInt(input);
            if (number < minNumber) {
                minNumber = number;
            }
            input = scanner.nextLine();
        }
        System.out.println(minNumber);
    }
}

File: JavaScript_Basic_04_2023_SolutionsOfProblems/05.LAB_PB-JS-While-Loop/P07.MinNumber.js
function minNumber(input) {
    let smallestNumber = Number.MAX_SAFE_INTEGER;
    let i = 0;
    while (input[i] !== "Stop") {
        let currentNumber = Number(input[i]);
        if (currentNumber < smallestNumber) {
            smallestNumber = currentNumber;
        }
        i++;
    }
    console.log(smallestNumber);
}
minNumber(["-10",
    "20",
    "-30",
    "Stop"])
